File: spring-pulsar-samples/java-sample/src/main/java/com/intuit/spring/pulsar/java/SampleApp.java
package com.intuit.spring.pulsar.java;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = {"com.intuit.spring.pulsar.sample01", "com.intuit.spring.pulsar.client"})
public class SampleApp {
    public static void main(String[] args) {
        SpringApplication.run(SampleApp.class, args);
    }
}

File: spring-pulsar-samples/kotlin-sample/src/main/kotlin/com/intuit/spring/pulsar/sample01/SampleApp.kt
package com.intuit.spring.pulsar.sample01

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackages = ["com.intuit.spring.pulsar.sample01", "com.intuit.spring.pulsar.client"])
class SampleApp

fun main() {
    SpringApplication.run(SampleApp::class.java)
}